packages/experience-docs/source/components/Icon/index.js

```diff
@@ -14,12 +14,15 @@ const getIcon = (icon) => find(ICONS, ({title}) => title === startCase(icon))
 const Icon = ({
   name,
   icon,
-  kind = 'flat'
+  kind = 'flat',
+  ...props
 }) => {
   const {src, title} = getIcon(icon)
-  return src && <SVGIcon kind={kind} svg={src} title={title} alt={title} />
+  return src && <SVGIcon {...props} kind={kind} svg={src} title={title} alt={title} />
```

Comment:
Fair enough. I did this quickly. I had always planned to explicitly add margin props.